zookeeper/init.sls

```diff
@@ -33,15 +33,8 @@ install-zookeeper:
     - group: root
     
 zookeeper-home-link:
-  alternatives.install:
-    - link: {{ zk.alt_home }}
-    - path: {{ zk.real_home }}
-    - onlyif: test -d {{ zk.real_home }} && test ! -L {{ zk.alt_home }}
-    - priority: 30
-    - require:
-      - archive: install-zookeeper
   file.symlink:
     - name: {{ zk.alt_home }}
     - target: {{ zk.real_home }}
```

Comment:
I am confused about the target..
As I understand `/etc/alternatives/zookeeper-home-link` links to the `zk.readl_home`. In case we use:

```
  file.symlink:
    - name: {{ zk.alt_home }}
    - target: /etc/alternatives/zookeeper-home-link
```

Then our link to old `real_home` will link to the link which goes again to old `real_home`. In this case firstly `file.symlink` state is unneeded and we won't be able to install a new version of zookeeper if `/etc/alternatives/zookeeper-home-link` is already exists. 

Maybe I just confused. I will try this scenario tomorrow.